### DISCUSSION TITLE
Server middleware rewrite (for custom domain) like NextJS

### DISCUSSION THREAD

**Discussion by zernonia** [nuxt/nuxt#16938] [Questions]
Hi, I'm looking at [Platforms Starter Kit](https://demo.vercel.pub/platforms-starter-kit) by Vercel, and was wondering how can I configure the server middleware just like the example [here](https://github.com/vercel/platforms/blob/main/pages/_middleware.ts) for custom domain.

Essentially, when user navigate to `foo.example.com`, it would shows content from `pages/_sites/[site]/index.vue`

**Comment by eMeRiKa13** [comment]
Hi @zernonia 

Did you find an answer for your question? I have the same problem.

Thanks

**Reply by zernonia** [reply]
Unfortunately, nope 🥲
